macos/README.md

```diff
@@ -30,12 +38,78 @@ GPU0:
 GPU1:
 ```
 
-Run the demo program:
+Run the demo program using the GPU:
+
+```console
+% kubectl exec pod/vulkan-demo-gpu -- sh -c 'time /vulkan_minimal_compute'
+
+real    0m0.357s
+user    0m0.317s
+sys     0m0.008s
+```
+
+Run the demo program using the CPU:
 
 ```console
-% kubectl exec pod/vulkan-demo -- sh -c 'time /vulkan_minimal_compute'
+% kubectl exec pod/vulkan-demo-cpu -- sh -c 'time /vulkan_minimal_compute'
+
+real    0m0.656s
+user    0m0.940s
+sys     0m0.012s
+```
 
-real    0m0.355s
-user    0m0.314s
-sys     0m0.016s
+## Running uVkCompute benchmarks
+
+To run this delete the vulkan demo pods and apply the uvkc pods:
```

Comment:
just give them the command to run

kubectl delete -f demo-gpu.yaml

btw how about we call that demo1 or give it some other name?
since both of them are demo